.ci/static-checks.sh

```diff
@@ -40,7 +40,7 @@ typeset -r arch_func_regex="_arch_specific$"
 repo=""
 specific_branch="false"
 force="false"
-branch=${branch:-main}
+branch=${branch:-stable-2.4}
```

Comment:
Same question, as I'm looking for a way to have done, maybe, by the script called on the jenkins side.